scripts/irods/test/test_iticket.py

```diff
@@ -888,3 +889,259 @@ def test_ticket_create_with_relative_paths_non_admin__issue_7103(self):
+        previous_ts = modify_ts
+        modify_ts, create_ts = get_modification_and_creation_time()
+        self.assertEqual(original_create_ts, create_ts)
+        self.assertGreater(modify_ts, create_ts)
+        self.assertGreater(modify_ts, previous_ts)
+
+        # Remove ticket host restriction
+        time.sleep(1)
+        self.admin.assert_icommand(['iticket', 'mod', ticket_string, 'remove', 'host', lib.get_hostname()])
+        previous_ts = modify_ts
+        modify_ts, create_ts = get_modification_and_creation_time()
+        self.assertEqual(original_create_ts, create_ts)
+        self.assertGreater(modify_ts, create_ts)
+        self.assertGreater(modify_ts, previous_ts)
+
+        # Add ticket user restriction
+        time.sleep(1)
+        self.admin.assert_icommand(['iticket', 'mod', ticket_string, 'add', 'user', self.admin.username])
+        previous_ts = modify_ts
+        modify_ts, create_ts = get_modification_and_creation_time()
+        self.assertEqual(original_create_ts, create_ts)
+        self.assertGreater(modify_ts, create_ts)
+        self.assertGreater(modify_ts, previous_ts)
+
+        # Remove ticket user restriction
+        time.sleep(1)
+        self.admin.assert_icommand(['iticket', 'mod', ticket_string, 'remove', 'user', self.admin.username])
+        previous_ts = modify_ts
+        modify_ts, create_ts = get_modification_and_creation_time()
+        self.assertEqual(original_create_ts, create_ts)
+        self.assertGreater(modify_ts, create_ts)
+        self.assertGreater(modify_ts, previous_ts)
+
+        # Add ticket group restriction
+        time.sleep(1)
+        self.admin.assert_icommand(['iticket', 'mod', ticket_string, 'add', 'group', 'public'])
+        previous_ts = modify_ts
+        modify_ts, create_ts = get_modification_and_creation_time()
+        self.assertGreater(modify_ts, previous_ts)
+
+        # Remove ticket group restriction
+        time.sleep(1)
+        self.admin.assert_icommand(['iticket', 'mod', ticket_string, 'remove', 'group', 'public'])
+        previous_ts = modify_ts
+        modify_ts, create_ts = get_modification_and_creation_time()
+        self.assertEqual(original_create_ts, create_ts)
+        self.assertGreater(modify_ts, create_ts)
+        self.assertGreater(modify_ts, previous_ts)
+
+    def test_ticket_mtime_is_updated_when_ticket_properties_are_modified_by_non_admins__issue_7125(self):
+
+        def get_modification_and_creation_time():
+            out, err, ec = self.admin.run_icommand(['iquest', '%s...%s', "select TICKET_MODIFY_TIME, TICKET_CREATE_TIME where TICKET_STRING = '{}'".format(ticket_string)])
+            self.assertEqual(ec, 0)
+            self.assertEqual(len(err), 0)
+
+            out = out.strip().split('...')
+            modification_time = out[0]
+            creation_time = out[1]
+
+            return int(modification_time), int(creation_time)
```

Comment:
This appears to be identical between the two tests. Could be broken out into a separate function for both to use. Up to you.

Reply:
Yeah I can do that.